.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+.idea/inspectionProfiles
```

Comment:
Why do we not want to share settings of this? Can't we agree on some settings and keep sharing this file?

Reply:
Hmmm, I am okay with adding this, but I think this just defines whether eslint is run by IntelliJ and errors are shown in the editor. I just figured that sometimes manual configuration is needed as automatic eslint config might not work for everybody.